src/libraries/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -34,6 +34,8 @@
              Link="Microsoft\IO\File.cs" />
     <Compile Include="$(CoreLibSharedDir)System\IO\FileInfo.cs"
              Link="Microsoft\IO\FileInfo.cs" />
+    <Compile Include="$(CoreLibSharedDir)System\IO\FileSystem.cs"
```

Comment:
Good point, no one has asked us for it. We added these dependencies because this project was already compiling FileSystemInfo. 
It looks to me that any API that we add to any type already compiling for MS.IO.Redist will be automatically released for that version as well. Is that desired? 

e.g: when `EnumerationOptions.MaxRecursionDepth` was added (https://github.com/dotnet/runtime/pull/48148), it was also added for MS.IO.Redist. 
Is that something that we should revert by ifdefing the code? If yes, then we should do a long-term solution to avoid hitting this problem.

cc @ericstj

Reply:
I just created https://github.com/dotnet/runtime/issues/54919 for discussing this.